### PR TITLE
Add session as mention in issue #20

### PR DESCRIPTION
**Pull request by krahser**
Here is the code for use flask-security with the [declarative SQLAlchemy](http://flask.pocoo.org/docs/0.12/patterns/sqlalchemy/#declarative).

Please let me know if there is something that I could do in a better way.

Thanks for your work.

### CODE REVIEW

File: flask_security/datastore.py [hunk 1]
```diff
@@ -255,6 +255,35 @@ def find_role(self, role):
...
+        SQLAlchemyUserDatastore.__init__(self,
+                                         PretendFlaskSQLAlchemyDb(session),
+                                         user_model,
+                                         role_model)
+
+    def commit(self):
+        # Old flask-sqlalchemy adds this weird attribute for tracking
+        # to Session. flask-sqlalchemy 2.0 does things more nicely.
+        try:
+            self.db.session.commit()
```
Comment (jirikuncar) [inline flask_security/datastore.py]: What about using `super(SQLAlchemySessionUserDatastore, self).commit()` instead on both places in this method?

File: setup.py [hunk 2]
```diff
@@ -24,6 +24,7 @@
     'pytest-flakes>=1.0.1',
     'pytest-pep8>=1.0.6',
     'pytest>=3.0.5',
+    'sqlalchemy>=1.1.6',
```
Comment (jirikuncar) [inline setup.py]: Is 1.1.6 really the minimal version of SQLAlchemy we can run?

File: flask_security/datastore.py (hunk 1 above)
Comment (krahser) [inline flask_security/datastore.py]: Yes! You are rigth that would be better.

File: setup.py (hunk 2 above)
Comment (krahser) [inline setup.py]: I don't find a proper way to test the minimal version, so I do it by hand. If you could give some tip about this it would be awesome.

**Pull request merged**